# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/external-memory/src/ptr.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/external-memory/src/ptr.rs b/third_party/rust/external-memory/src/ptr.rs
deleted file mode 100644
--- a/third_party/rust/external-memory/src/ptr.rs
+++ /dev/null
@@ -1,20 +0,0 @@
-#[derive(Debug)]
-/// Pointer to a host allocated memory.
-pub struct Ptr(*mut std::ffi::c_void);
-impl Ptr {
-    /// Get the inner ptr
-    pub fn as_raw_ptr(&self) -> *mut std::ffi::c_void {
-        self.0
-    }
-}
-impl<T> From<*mut T> for Ptr {
-    fn from(ptr: *mut T) -> Self {
-        Self(unsafe { std::mem::transmute(ptr) })
-    }
-}
-impl std::ops::Deref for Ptr {
-    type Target = *mut std::ffi::c_void;
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}